src/lib/components/Form/Inputs/MultiSelectInput.svelte

```diff
@@ -0,0 +1,141 @@
+<script lang="ts">
+  import Chip, { Set as ChipSet, Text, TrailingAction } from '@smui/chips';
+  import Autocomplete from '@smui-extra/autocomplete';
+  import type { Option } from '$lib/models/form';
+  import type { DynamicFieldConfig, FieldConfig, ValidationResult } from '../FieldsConfig.js';
+  import FieldHint from '../FieldHint.svelte';
+
+  type InputProps = {
+    onChange?: (value: string[]) => void;
+    value?: string[];
+    class?: string;
+    label?: string;
+    fieldConfig?: FieldConfig<string> | DynamicFieldConfig;
+    options: Option[];
+    validationResult?: ValidationResult;
+    disabled?: boolean;
+  };
+
+  let {
+    onChange,
+    value = [],
+    label,
+    class: wrapperClass,
+    fieldConfig,
+    options,
+    validationResult
+  }: InputProps = $props();
+
+  let element: HTMLFieldSetElement;
+  let inputValue = $state<string>();
+
+  const chips = $derived<Option[]>(
+    (value
+      ?.map((val) => options.find((option) => option.key === val))
+      .filter(Boolean) as Option[]) || []
+  );
+
+  const filteredOptions = $derived<Option[]>(
+    options.filter((option) => !value?.includes(option.key))
+  );
+
+  const onSelect = (event: CustomEvent<Option>) => {
+    event.preventDefault();
+
+    // we don't want to show the selection in the input field
+    inputValue = '';
+
+    // TODO: list does not close on selection. workaround needed
```

Comment:
Is this needed for this PR?

Reply:
It would be nice, but i already spent some hours on trying to get this working. Functionality is still completly given.